src/Str.php

```diff
@@ -76,28 +84,27 @@ public static function asClassName(string $value, string $suffix = ''): string
      */
     public static function asTwigVariable(string $value): string
     {
-        $value = trim($value);
-        $value = preg_replace('/[^a-zA-Z0-9_]/', '_', $value);
-        $value = preg_replace('/(?<=\\w)([A-Z])/', '_$1', $value);
-        $value = preg_replace('/_{2,}/', '_', $value);
-        $value = strtolower($value);
-
-        return $value;
+        return u($value)
```

Comment:
This can be simplified as `return u($value)->snake();`

Reply:
Yep